.circleci/config.yml

```diff
@@ -1,256 +1,27 @@
-defaults: &defaults
```

Comment:
Reminder: You'll need to revert this before we can merge this in.

I assume you were doing this to try to get the builds going in your fork on CircleCI. If that is the case, try the following config:

```
defaults: &defaults
  docker:
    - image: 087285199408.dkr.ecr.us-east-1.amazonaws.com/circle-ci-test-image-base:go1.13

version: 2.0
jobs:
  test:
    <<: *defaults
    steps:
      - checkout

      - run:
          name: run tests
          command: |
            mkdir -p /tmp/logs
            run-go-tests --path test --timeout 2h | tee /tmp/logs/all.log
          no_output_timeout: 3600s

      - run:
          command: terratest_log_parser --testlog /tmp/logs/all.log --outputdir /tmp/logs
          when: always
      - store_artifacts:
          path: /tmp/logs
      - store_test_results:
          path: /tmp/logs

workflows:
  version: 2
  build-and-test:
    jobs:
      - test:
          filters:
            tags:
              only: /^v.*/
```

Reply:
Yeah that's the goal test the fork circleci compilation

Reply:
![image](https://user-images.githubusercontent.com/13622697/79201152-e8aa5700-7e37-11ea-8767-153156789874.png)
@yorinasub17 I'm still trying to test the build but now I receiving this error I could use circleci/golang:1.13 but maybe I need configuration for **run-go-tests**

Reply:
Error using circleci/golang:1.13

![image](https://user-images.githubusercontent.com/13622697/79201342-38891e00-7e38-11ea-9fd2-fcf531fd2383.png)

Reply:
Ah I thought our image was public, but it is not :-/

In any case, a lot of the scripts we use are in our closed source library. I recommend running just the tests you added locally, and once you are confident, we can kick off the build ourselves (we have a script that pulls down the current snapshot of forks into our repo so circle builds it). We can only do this if you back out the circleci config, so as soon as you roll that back, I can kick off a build.